code-challenges.test.js

```diff
@@ -48,9 +87,26 @@ const studyMinutesWeek2 = {
   thursday: 20,
   friday: 15,
   saturday: 65
-}
+};
+
+const result = sortObjectValues(studyMinutesWeek2);
+// This correctly sorts the value for our week2
+const expected = [10, 15 , 20 , 45, 60, 65 ,100];
+// expected array value that we want to be the outcome
+expect(result).toEqual(expected);
+});
+});  
+  
 // Expected output: [10, 15, 20, 45, 60, 65, 100]
 
 // b) Create the function that makes the test pass.
 
-// Pseudo code:
+function sortObjectValues(obj) {
+  // The function take the object and extracts its values
+  const values = Object.values(obj);
+  //it returns those values in ascending order with the .sort method
+  return values.sort((a, b) => a - b);
+}
+
+module.exports = { sortObjectValues };
+
```

Comment:
I am curious what this line is doing for you`module.exports = { sortObjectValues };`?